src/lib/evil/evil_unistd.h

```diff
@@ -30,9 +30,10 @@ EVIL_API int ftruncate(int fd, off_t size);
 #define X_OK    0       /* execute permission, originally '1', just a bypass here*/
 #define F_OK    0       /* Test for existence.  */
 
-#define STDIN_FILENO __fileno(stdin)
-#define STDOUT_FILENO __fileno(stdout)
-#define STDERR_FILENO __fileno(stderr)
+#include <stdio.h>
```

Comment:
maybe add it in the beginning of the file with other includes?

Reply:
Yes, you're right. Didn't see that it wasn't the beginning of the file.

Reply:
done